generators/app/templates/infrastructure/src/components/menu/CollapsibleMenuItem.js

```diff
@@ -0,0 +1,30 @@
+import React, { useCallback, useState } from "react";
+import PropTypes from "prop-types";
+import { Collapse, List } from "@material-ui/core";
+import MenuItem from "./MenuItem";
+
+const CollapsibleMenuItem = ({ menu, drawerOpen, activeRoute }) => {
+  const [open, setOpen] = useState(false)
+  const toggleSubMenu = useCallback(() => setOpen(current => !current), [])
+
+  return (
+    <>
+      <MenuItem menu={menu} drawerOpen={drawerOpen} subMenuOpen={open} onToggleSubMenu={toggleSubMenu} />
+      <Collapse in={open} timeout="auto" unmountOnExit>
+        <List disablePadding>
+          {menu?.children?.map((subMenu, key) => (
```

Comment:
I think you should add an example for a menu with aggregated children in menuConfig file